### PR TITLE
Issue #636 - Fix all labels in drop lists 'module tree' and 'fields' in AOR_Reports

### PR DESCRIPTION
**Pull request by shogunpol**
All labels in drop lists 'module tree' and 'fields' in AOR_Reports are fixed
## Description

This fix fixes all labels for AOR_Reports
References #636

## Motivation and Context

It added missing labels in language files and add missing variables in vardef files for related modules.
## How To Test This

The hanges has ben tested by me only in Suite P module.

@shogunpol ^^^ Need to provide better Steps to be able to Test.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/AM_TaskTemplates/language/en_us.lang.php [hunk 1]
```diff
@@ -80,4 +80,6 @@
     'LBL_UTILIZATION' => 'Utilization (%)',
     'LBL_DURATION' => 'Duration (days)',
     'LBL_AM_TASKTEMPLATES_AM_PROJECTTEMPLATES_FROM_AM_PROJECTTEMPLATES_TITLE' => 'Project Templates',
+
+    'LBL_AM_TASKTEMPLATES_AM_PROJECTTEMPLATES_FROM_AM_TASKTEMPLATES_TITLE' => 'Task Templates: Project Templates from Task Templates Title',
```
Comment (horus68) [inline modules/AM_TaskTemplates/language/en_us.lang.php]: maybe this is a too long title?

Comment (shogunpol) [inline modules/AM_TaskTemplates/language/en_us.lang.php]: No, is exact long as suppose to be, also if @horus68 found just one sting he is not happy, that means to me i did good job. Expect star in my profile :) or at least medal :)

Comment (horus68) [inline modules/AM_TaskTemplates/language/en_us.lang.php]: a medal from me! (also look into my other comments on this PR!)

File: modules/AOS_Invoices/language/en_us.lang.php [hunk 2]
```diff
@@ -201,6 +201,11 @@
     'LBL_SHIPPING_TAX_USDOLLAR' => 'Shipping Tax (Default Currency)',
     'LBL_SHIPPING_TAX_AMT_USDOLLAR' => 'Shipping Tax (Default Currency)',
     'LBL_GRAND_TOTAL_USDOLLAR' => 'Grand Total (Default Currency)',
-    'LBL_INVOICE_TO' => 'Invoice To'
+    'LBL_INVOICE_TO' => 'Invoice To',
+    'LBL_DISCOUNT__AMOUNT_USDOLLAR' => 'Discount Amount (Default Currency)',
```
Comment (shogunpol) [inline modules/AOS_Invoices/language/en_us.lang.php]: In this PR i did not changed at all existed LBL pointers, please raise suggestion in separate issue.

File: modules/ProspectLists/language/en_us.lang.php [hunk 3]
```diff
@@ -95,7 +95,11 @@
   'LBL_DOMAIN_NAME'=>'Domain Name',
   'LBL_DOMAIN'=>'No emails to Domain',
   'LBL_LIST_PROSPECTLIST_NAME'=>'Name',
-	'LBL_MORE_DETAIL' => 'More Detail' /*for 508 compliance fix*/,
+  'LBL_MORE_DETAIL' => 'More Detail' /*for 508 compliance fix*/,
+
+  'LBL_EMAIL_MARKETING'=>'Email Marketing',
+  'LBL_EMAIL_MARKETING'=>'Email Marketing',
```
Comment (shogunpol) [inline modules/ProspectLists/language/en_us.lang.php]: Fixed

File: modules/AOS_Invoices/language/en_us.lang.php (hunk 2 above)
Comment (samus-aran) [inline modules/AOS_Invoices/language/en_us.lang.php]: That's a good point.  We would need to change it in the vardefs for AOS_Invoicesvardefs .  Perhaps a separate PR for that.

File: modules/AM_ProjectTemplates/language/en_us.lang.php [hunk 4]
```diff
@@ -142,5 +142,8 @@
 	'LBL_SCHEDULING_FORM_TITLE' => 'Resource List',
 	'LBL_NONE' => 'None',
 
+	'LBL_AOS_PRODUCTS_AM_PROJECTTEMPLATES_1_FROM_AM_PROJECTTEMPLATES_TITLE' => 'Product: Project Templates from Project Templates Title',
+	'LBL_AM_PROJECTTEMPLATES_PROJECT_1_FROM_PROJECT_TITLE' => 'Project Templates Project from Project Title',
```
Comment (horus68) [inline modules/AM_ProjectTemplates/language/en_us.lang.php]: @shogunpol To be the same as in modules/Projects/ the value should have a` : ` as a separator
'LBL_AM_PROJECTTEMPLATES_PROJECT_1_FROM_PROJECT_TITLE' => 'Project Templates: Project from Project Title',

File: modules/AOS_Quotes/language/en_us.lang.php [hunk 5]
```diff
@@ -201,5 +201,12 @@
     'LBL_SHIPPING_TAX_AMT_USDOLLAR' => 'Shipping Tax (Default Currency)',
     'LBL_GRAND_TOTAL_USDOLLAR' => 'Grand Total (Default Currency)',
     'LBL_QUOTE_TO' => 'Quote To',
+
+    'LBL_SUBTOTAL_TAX_AMOUNT_USDOLLAR' => 'Subtotal Tax (Default Currency)',
```
Comment (horus68) [inline modules/AOS_Quotes/language/en_us.lang.php]: is this a subtotal `of` tax only or `Subtotal + Tax` ?
In modules/AOS_Line_Item_Groups/language/en_us.lang.php the string is:
`'LBL_SUBTOTAL_TAX_AMOUNT_USDOLLAR' => 'Subtotal + Tax (Default Currency)',`

File: modules/AM_ProjectTemplates/language/en_us.lang.php (hunk 4 above)
Comment (shogunpol) [inline modules/AM_ProjectTemplates/language/en_us.lang.php]: @horus, fair point, they may be different, i tried to keep everything consistent as possible, however sometimes you can see some differences, both label strings are descriptive enough. Also they totally independent in each module so you don't have to worry about malfunctioning.

File: modules/AOS_Quotes/language/en_us.lang.php (hunk 5 above)
Comment (shogunpol) [inline modules/AOS_Quotes/language/en_us.lang.php]: Is Subtotal + Tax.

File: modules/AM_ProjectTemplates/language/en_us.lang.php (hunk 4 above)
Comment (horus68) [inline modules/AM_ProjectTemplates/language/en_us.lang.php]: not a code issue... only a translator issue.
On Crowdin we will only have to translate one sentence as the other get auto-filled (if they are 100% the same)

File: modules/AOS_Quotes/language/en_us.lang.php (hunk 5 above)
Comment (horus68) [inline modules/AOS_Quotes/language/en_us.lang.php]: Can you change it to `Subtotal + Tax (Default Currency) `?
Just to avoid translators questions!

Comment (shogunpol) [inline modules/AOS_Quotes/language/en_us.lang.php]: Yes, can be handle in when you create new issue/suggestion. This one is merged.

**Pull request merged**